src/main/java/com/synopsys/integration/blackduck/nexus3/task/inspector/InspectorMetaDataProcessor.java

```diff
@@ -37,11 +37,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.synopsys.integration.blackduck.api.generated.component.RiskCountView;
-import com.synopsys.integration.blackduck.api.generated.component.VersionBomOriginView;
-import com.synopsys.integration.blackduck.api.generated.enumeration.PolicySummaryStatusType;
+import com.synopsys.integration.blackduck.api.generated.component.ComponentVersionRiskProfileRiskDataCountsView;
+import com.synopsys.integration.blackduck.api.generated.enumeration.PolicyStatusType;
+import com.synopsys.integration.blackduck.api.generated.view.ProjectVersionComponentView;
 import com.synopsys.integration.blackduck.api.generated.view.ProjectVersionView;
-import com.synopsys.integration.blackduck.api.generated.view.VersionBomComponentView;
+import com.synopsys.integration.blackduck.api.manual.throwaway.generated.component.VersionBomOriginView;
```

Comment:
This may be unrelated to nexus, but using something in a throwaway package doesn't seem like a good idea

Reply:
I agree but ProjectVersionComponentView.getOrigins() returns that throwaway object. So until blackduck-common-api is updated we are stuck with this.